### PR TITLE
Changes for Delete Message API from beta to V1.0

### PR DESCRIPTION
**Pull request by sumitgupta3**
Public Docs update for taking the Delete Message APIs from beta to V1.0

### CODE REVIEW

File: api-reference/v1.0/api/chatmessage-softdelete.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+title: "chatMessage: softDelete"
+description: "Delete a single message or message reply in a channel or a chat."
+author: "Ramjot Singh"
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/chatmessage-softdelete.md:3]: Line 3: Please put here the github alias of the main contact to report any issues or questions related to this document.

File: api-reference/v1.0/api/chatmessage-undosoftdelete.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
+---
+title: "chatMessage: undoSoftDelete"
+description: "Undelete a single message or a message reply in a channel or a chat."
+author: "Ramjot Singh"
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/chatmessage-undosoftdelete.md:4]: Line 4: Please put here the github alias of the main contact to report any issues or questions related to this document.

**Review by isvargasmsft — APPROVED** [review]
Hello @sumitgupta3, 

Thank you very much for your work in this PR. 

Just a small ask: before adding the label "ready to merge" please confirm the author field in the file metadata has the right value.

Thanks!

File: api-reference/v1.0/api/chatmessage-softdelete.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
+---
+title: "chatMessage: softDelete"
+description: "Delete a single message or message reply in a channel or a chat."
+author: "Ramjot Singh"
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/chatmessage-softdelete.md:4]:
```suggestion
author: "RamjotSingh"
```

File: api-reference/v1.0/api/chatmessage-undosoftdelete.md (hunk 2 above)
Comment (isvargasmsft) [inline api-reference/v1.0/api/chatmessage-undosoftdelete.md:4]:
```suggestion
author: "RamjotSingh"
```

**Comment by Lauragra** [comment]
@sumitgupta3 , if you would like to publish this PR, please remove the **do not merge** label and add **ready to merge**.

**Pull request merged**